pkg/server/datastore/sqlstore/sqlstore_test.go

```diff
@@ -806,6 +811,7 @@ func (s *PluginSuite) TestListAttestedNodes() {
 			nodes:               []*common.AttestedNode{nodeA, nodeB, nodeC, nodeD, nodeE},
 			byAttestationType:   "T1",
 			bySelectors:         bySelectors(datastore.Subset, "S1"),
+			byCanReattest:       &canReattestFalse,
```

Comment:
why byCanReattest is required here? may you move this byCanReattest to `all filters` instead?

Reply:
Sure